server/routes/visitJourney/visitJourneyUtils.ts

```diff
@@ -26,6 +26,18 @@ export const validationErrorsToMoJAlert = (
     }
   }
 
+  if (validationErrors.includes('APPLICATION_INVALID_VISIT_DATE_BLOCKED')) {
+    return {
+      mojAlert: {
+        title: 'This visit date has been blocked.',
+        text: 'Select a new visit time.',
+        variant: 'warning',
```

Comment:
```suggestion
        variant: 'error',
```

Should this be an 'error' type (red) because the user cannot proceed: they have to select a new session?

(Compare to the `warning` one for no slot capacity: orange, signifying user can proceed with caution.)